src/Admin/AbstractAdmin.php

```diff
@@ -2584,19 +2571,16 @@ public function getDashboardActions()
         return $filter[$name] === $default[$name];
     }
 
-    /**
-     * Check object existence and access, without throw Exception.
-     *
-     * @param string $action
-     * @param object $object
-     *
-     * @return bool
-     */
-    public function canAccessObject($action, $object)
+    public function canAccessObject(string $action, object $object): bool
     {
         return $object && $this->id($object) && $this->hasAccess($action, $object);
```

Comment:
`$object` cannot be evaluated as false anymore.
```suggestion
        return $this->id($object) && $this->hasAccess($action, $object);
```